backends/kv/reader.go

```diff
@@ -148,6 +149,19 @@ func (ki *Iterator) Next() bool {
 		byName[field.Name] = col
 	}
 
+	if specificColumnsRequested {
+		for _, attr := range systemAttrs {
+			if containsString(ki.request.Proto.Columns, attr) {
+				sysCol, err := frames.NewSliceColumn(attr, make([]int64, 0))
```

Comment:
Are we sure all of the system attribute are/will be of type int64?

Reply:
yes, they are all int64 except __name (verified with ortal)